### PR TITLE
Added artifact sanitizing if it is secured again clicking

### PR DESCRIPTION
**Pull request by Billith**
Use case: selecting IOCs from blog/articles where they are secured again clicking (eg. `127[.]0.0.1` or `hxxps://localhost[.]local`)

### CODE REVIEW

File: firefox/background.js [hunk 1]
```diff
@@ -552,19 +552,18 @@ var url = "";
 //create empty artifact variable
 var artifact = "";
 
-function sanitizeArtifact(artifact) { 
-    while(String(artifact).includes("[.]")) {
-            artifact = String(artifact).replace("[", "").replace("]", "");
+function sanitizeArtifact(artifact) {
+    while(artifact.includes("[.]")) {
+        artifact = artifact.replace("[.]", ".");
     }
```
Comment (mitchmoser) [inline firefox/background.js:559]: Let's change these replace statements below to `hxxp://` and `hxxps://`

File: chrome/background.js [hunk 2]
```diff
@@ -419,19 +419,18 @@ function copyStringToClipboard(str) {
     document.body.removeChild(el);
     }
 
-function sanitizeArtifact(artifact) { 
-    while(String(artifact).includes("[.]")) {
-            artifact = String(artifact).replace("[", "").replace("]", "");
+function sanitizeArtifact(artifact) {
+    while(artifact.includes("[.]")) {
+        artifact = artifact.replace("[.]", ".");
     }
```
Comment (mitchmoser) [inline chrome/background.js:426]: Let's change these replace statements below to `hxxp://` and `hxxps://`

File: chrome/background.js [hunk 3]
```diff
@@ -435,6 +451,10 @@ chrome.contextMenus.onClicked.addListener((info, tab) => {
         var src = new URL(info.srcUrl);
         artifact = src.host;
     }
+
+    // sanitize artifact if it is secured agains clicking
```
Comment (mitchmoser) [inline chrome/background.js]: typo:
    // unsanitize artifact if it is secured against clicking

File: firefox/background.js [hunk 4]
```diff
@@ -571,6 +587,9 @@ browser.contextMenus.onClicked.addListener((info, tab) => {
...
     // will copy the selection to clipboard after switch statement
 
+    // sanitize artifact if it is secured agains clicking
```
Comment (mitchmoser) [inline firefox/background.js]: typo:
    // unsanitize artifact if it is secured against clicking

**Review by mitchmoser — COMMENTED** [review]
Thanks for the pull request! I understand the use case and this solution is easy to follow.
I made a couple of notes for minor changes to be made before approving and merging the request.

File: firefox/background.js (hunk 1 above)
Comment (Billith) [reply firefox/background.js:559]: I haven't thought about cases where "hxxp" or "hxxps" can be in domain or path. Of course I'm gonna add that.

File: chrome/background.js (hunk 2 above)
Comment (Billith) [reply chrome/background.js:426]: I haven't thought about cases where "hxxp" or "hxxps" can be in domain or path. Of course I'm gonna add that.

**Pull request merged**